app/src/main/java/org/wikipedia/feed/onthisday/OnThisDayFragment.java

```diff
@@ -338,17 +336,20 @@ public void setFields(@NonNull final OnThisDay.Event event) {
             setPagesViewPager(event);
         }
 
+        @SuppressWarnings("checkstyle:magicnumber")
         private void setPagesViewPager(OnThisDay.Event event) {
             if (event.pages() != null) {
                 ViewPagerAdapter viewPagerAdapter = new ViewPagerAdapter(getChildFragmentManager(), event.pages(), wiki);
                 pagesViewPager.setAdapter(viewPagerAdapter);
                 pagesViewPager.setOffscreenPageLimit(2);
-                 new TabLayoutMediator(pagesIndicator, pagesViewPager, (tab, position) -> { }).attach();
                 pagesViewPager.setVisibility(VISIBLE);
-                pagesIndicator.setVisibility(event.pages().size() == 1 ? GONE : VISIBLE);
+                if (event.pages().size() > 1) {
+                    pagesViewPager.addItemDecoration(new IndicatorDotDecor(DimenUtil.roundedDpToPx(4),
+                            DimenUtil.roundedDpToPx(8), 75, ResourceUtil.getThemedColor(getContext(), R.attr.chart_shade5),
+                            ResourceUtil.getThemedColor(getContext(), R.attr.colorAccent)));
```

Comment:
Do you think it is a good idea to have a default style set in the `IndicatorDotDecor` because the `OnThisDayFragment` and `NewsFragment` are having the same style of `IndicatorDotDecor`?

Reply:
good idea... will do that